plumed/build.sh

```diff
@@ -18,6 +18,10 @@ export CXXFLAGS="${CXXFLAGS//-O2/-O3}"
 
 # libraries are explicitly listed here due to --disable-libsearch
 export LIBS="-lboost_serialization -lfftw3 -lgsl -lgslcblas -llapack -lblas -lz $LIBS"
+export LIBS="-lmetatensor_torch -lmetatensor -ltorch -lc10 -ltorch_cpu $LIBS"
```

Comment:
@Luthaf any specific reason for removing libraries here (boost, fftw3, gslm etc)? I suspect they won't be linked

Reply:
I did not remove anything, I added another line below. The full set of libraries will be `-lmetatensor_torch -lmetatensor -ltorch -lc10 -ltorch_cpu -lboost_serialization -lfftw3 -lgsl -lgslcblas -llapack -lblas -lz`, plus whatever was in `$LIBS` before these two lines

Reply:
Sorry I didn't read correctly the script, it's fine as you did